src/Appwrite/Detector/Detector.php

```diff
@@ -47,7 +47,18 @@ public function getOS(): array
      */
     public function getClient(): array
     {
-        $client = $this->getDetector()->getClient();
+        if (strpos($this->userAgent, 'Terminal') !== false) {
```

Comment:
Not sure if `Terminal` is the right User Agent to use here 🤔 

Especially since there are User Agents out there using the term `KioskTerminal` => https://user-agents.net/string/mozilla-5-0-windows-nt-6-2-wow64-applewebkit-537-36-khtml-like-gecko-chrome-63-0-3239-132-safari-537-36-kioskterminal-2-12-0

We could be very specific here and use `AppwriteCLI`?

Reply:
I think this is a good idea. Maybe instead of `AppwriteCLI` => `Appwrite CLI`

Reply:
Agree!

Reply:
Done 👍